src/components/validators/project-name-available-validator.js

```diff
@@ -28,6 +31,10 @@
 
             return deferred.promise;
           };
+
+          scope.$watch("organizationId", function() {
+            ngModel.$validate();
```

Comment:
No `$validateAsync()` exists, but is covered through the [`validate()`](https://docs.angularjs.org/api/ng/type/ngModel.NgModelController#$validate) method:

> Runs each of the registered validators (first synchronous validators and then asynchronous validators).